tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -218,13 +225,28 @@ def check_meshcop_service_by_data(self, br, service_data):
         self.assertEqual(service_data['txt']['rv'], '1')
         self.assertIn(service_data['txt']['tv'], ['1.1.0', '1.1.1', '1.2.0', '1.3.0'])
 
-    def discover_all_meshcop_services(self, host):
-        instance_names = host.browse_mdns_services('_meshcop._udp')
+    def discover_all_meshcop_services(self, host, type):
+        instance_names = host.browse_mdns_services(type)
         services = []
         for instance_name in instance_names:
-            services.append(host.discover_mdns_service(instance_name, '_meshcop._udp', None))
+            services.append(host.discover_mdns_service(instance_name, type, None))
         return services
 
+    def check_meshcop_e_service(self, br, host, status):
+        lifetime = 500000
+
+        if (status == False):
+            return
+
+        ephemeral_key = br.start_ephemeral_key_mode(lifetime)
+
+        self.assertEqual(len(ephemeral_key), 9)
+        self.assertEqual('active', br.get_ephemeral_key_state())
```

Comment:
Better to keep the consistency of parameters' order.

Reply:
Accept, thanks!